apps/autoscale-by-metric.html.markerb

```diff
@@ -0,0 +1,181 @@
+---
+title: Autoscale based on metrics
+layout: docs
+nav: firecracker
+---
+
+Most applications on Fly.io can use the request-based autoscaling that is built into our proxy by setting [the soft & hard limits in your fly.toml](/docs/reference/configuration/#http_service-concurrency). However, some applications need to scale on other metrics besides requests such as a work queue depth. In these instances, you can use our open source [fly-autoscaler](https://github.com/superfly/fly-autoscaler) to scale your workload.
```

Comment:
@benbjohnson Suggest starting with what it is/does and what apps it's good for. Then say that most apps can use auto start and stop. Something like this but with your better understanding/edits:

```suggestion
The metrics-based autoscaler scales an app's Machines based on any metric, such as pending work items or queue depth. Scaling on metrics other than requests is useful for apps that aren't running web services, like background workers, for example. Apps with services that need to scale based on HTTP requests can use the [auto stop and start feature](https://fly.io/docs/apps/autostart-stop/) for Machines that's built into the Fly Proxy.
```